Rendering failure with math mode in <latex> in cloze cards

While running the tests, I noticed the following test failure in TestCloze.test_latex:

<details>
<summary>Error output</summary>

```
======================================================================
FAIL: test_cloze.TestCloze.test_latex
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/jakob/Documents/development/forks/mnemosyne/tests/test_cloze.py", line 411, in test_latex
    assert "<img src" in card.answer()
AssertionError: 
-------------------- >> begin captured stdout << ---------------------
Command `latex -interaction=nonstopmode tmp.tex` failed with rc=1
Command output:
This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2021/Arch Linux) (preloaded format=latex)
 restricted \write18 enabled.
entering extended mode
(./tmp.tex
LaTeX2e <2020-10-01> patch level 4
L3 programming layer <2021-02-18>
(/usr/share/texmf-dist/tex/latex/base/article.cls
Document Class: article 2020/04/10 v1.4m Standard LaTeX document class
(/usr/share/texmf-dist/tex/latex/base/size12.clo))
(/usr/share/texmf-dist/tex/latex/l3backend/l3backend-dvips.def)
No file tmp.aux.

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              

! LaTeX Error: \expandafterallowed only in math mode.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4 \mathbf{F}
              
(That makes 100 errors; please try again.)
No pages of output.
Transcript written on tmp.log.

Command input:
\documentclass[12pt]{article}
\pagestyle{empty}
\begin{document}
\mathbf{F}
\end{document}

Command `dvipng -D 200 -T tight tmp.dvi` failed with rc=1
Command output:
tmp.dvi: No such file or directory
This is dvipng 1.17 Copyright 2002-2015, 2019 Jan-Ake Larsson


--------------------- >> end captured stdout << ----------------------
```

</details>

This error is caused by the input text on this line:

https://github.com/mnemosyne-proj/mnemosyne/blob/5941cd88a35e011758c70a48a3a7ffd2f9268c67/tests/test_cloze.py#L408

 Basically, the TeX file fails to compile because `\mathbf` is used outside of math mode. I’m actually surprised that it ever worked – I guess it might have been a warning (rather than an fatal error) in earlier versions of LaTeX? Regardless, I believe the test case should be updated – probably either by replacing the text with something that doesn't use math mode commands, or by surrounding the string with `$…$` and teaching cloze cards to detect math mode when extracting fragments from `<latex>` elements.

I could do the former change myself (if such a trivial change is even worth the back and forth), but if the latter solution is more desirable I’d rather leave it to someone else, since this is a feature that I don’t actually use myself (I just happened to notice the test failure).

## Software:

- Fully up-to-date Arch Linux
- TeX Live 2021 from the Arch repos
- Python 3.9.5
- Mnemosyne 5941cd88a35e011758c70a48a3a7ffd2f9268c67

--- a/tests/test_cloze.py
+++ b/tests/test_cloze.py
@@ -405,7 +405,7 @@ third in 2008"""}
                                           grade=-1, tag_names=["default"])[0]
         assert "<img src" in card.answer()
 
-        fact_data = {"text": """<latex>[\mathbf{F}]=[q]([\mathbf{E}]+[\mathbf{v}\times\mathbf{B}]</latex>"""}
+        fact_data = {"text": """<latex>[F]=[q]([E]+[v\times B]</latex>"""}
         card = self.controller().create_new_cards(fact_data, card_type,
                                           grade=-1, tag_names=["default"])[0]
         assert "<img src" in card.answer()

